ArchiSteamFarm/Core/NativeMethods.cs

```diff
@@ -26,6 +28,12 @@
 namespace ArchiSteamFarm.Core;
 
 internal static partial class NativeMethods {
+	[DefaultDllImportSearchPaths(DllImportSearchPath.System32)]
+	[SupportedOSPlatform("Windows")]
+	[return: MarshalAs(UnmanagedType.Bool)]
```

Comment:
Misc: Do we need that return if we're using `void` anyway?

Reply:
Yes.